easybuild/easyconfigs/r/rclone/rclone-1.54.1-amd64.eb

```
@@ -0,0 +1,35 @@
+# Authors:: Jack Perdue <j-perdue@tamu.edu> - TAMU HPRC - https://hprc.tamu.edu
+
+easyblock = 'MakeCp'
+
+name = 'rclone'
+version = '1.54.1'
+versionsuffix = '-amd64'
+
+homepage = 'https://rclone.org/'
+
+description = """
+ Rclone is a command line program to sync files and directories to and from
+ a variety of online storage services
+"""
+
+toolchain = SYSTEM
+
+source_urls = ['https://downloads.%(name)s.org/v%(version)s/']
+sources = ['%(name)s-v%(version)s-linux%(versionsuffix)s.zip']
+checksums = ['c52cbf3646a2d15765b87cf05fc3b2bca3b1d2782d4922046c597bd979e42720']
+
+skipsteps = ['build']
+
+files_to_copy = [
+    (['rclone'], 'bin'),
+    (['rclone.1'], 'share/man/man1'),
+    (['README.*'], 'share/doc'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/rclone', 'share/doc/README.txt', 'share/man/man1/rclone.1'],
+    'dirs': []
+}
```

can you add a sanity check command here?
```suggestion
}

sanity_check_command = ['rclone --help']
```